src/app/user/user.service.ts

```diff
@@ -15,7 +15,7 @@ import { User } from '../models/user';
 export class UserService {
   private headers = new Headers({'Content-Type': 'application/json'});
   private userUrl = process.env.API_URL + 'user';  // URL to web api
-  private identitiesUrl = process.env.API_URL + 'identities';  // URL to web api
+  private usersUrl = process.env.API_URL + 'users';  // URL to web api
```

Comment:
Not yet. And as I said in the description (see above ^) please do NOT merge until almighty/almighty-core#692 is merged ;)